atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -223,6 +169,69 @@ def _send_mail(self, receivers_list, subject, body):
             if s is not None:
                 s.quit()
 
+    def _get_email_from_koji_obj(self, obj):
+        if obj.get('krb_principal') and obj['krb_principal'] != '':
+            return obj['krb_principal'].lower()
+        else:
+            return '@'.join([obj['owner_name'], self.email_domain])
```

Comment:
`error_address` should be OSBS admins (read: us), as it means the plugin has crashed / misconfigured.

This would be configured in osbs-client